clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -286,11 +289,17 @@ pt2ir_t::convert(DR_PARAM_IN const uint8_t *pt_data, DR_PARAM_IN size_t pt_data_
 
     /* This flag indicates whether manual synchronization is required. */
     bool manual_sync = true;
-
     /* PT raw data consists of many packets. And PT trace data is surrounded by Packet
      * Stream Boundary. So, in the outermost loop, this function first finds the PSB. Then
      * it decodes the trace data.
      */
+    uint64_t decoded_instr_count = 0;
+    uint64_t recoverable_error_count = 0;
+    /* XXX: This is currently set based on empirical observations. We use this heuristic
+     * to detect recoverable errors: any non-consecutive error of type pte_bad_query, and
+     * errors up to this limit are attempted to recover from by retrying.
+     */
+    constexpr int kErrorLimit = 100;
```

Comment:
style: for DR it's all-caps for constants.

Reply:
Done